python/tests/tsutil.py

```diff
@@ -1002,6 +1002,19 @@ def cmp_edge(i, j, tables):
     return ret
 
 
+def cmp_migration(i, j, tables):
+    ret = tables.migrations.time[i] - tables.migrations.time[j]
```

Comment:
It doesn't really matter I'd imagine, but 
```
migration_i = tables.migrations[i]
migration_j = tables.migrations[j]
ret = migration_i.time - migration_j.time
# etc
```
would avoid repeatedly building and returning numpy arrays for each of the columns.